app-backend/project/Dependencies.scala

```diff
@@ -40,4 +40,5 @@ object Dependencies {
   val scalacacheCaffeine      = "com.github.cb372"            %% "scalacache-caffeine"               % Version.scalacache
   val elasticacheClient       = "com.amazonaws"                % "elasticache-java-cluster-client"   % Version.elasticacheClient
   val shapeless               = "com.chuusai"                 %% "shapeless"                         % Version.shapeless
+  val findbugAnnotations      = "com.google.code.findbugs"     % "annotations"                       % Version.findbugAnnotations % "compile"
```

Comment:
This library is needed (for some reason that I don't fully grasp), to allow for the compiler to understand certain annotations within the caching libraries at compile time.